.env.tpl

```diff
@@ -7,12 +7,9 @@ CLUSTER_BASIC_AUTH_TOKEN="???"
 # Indexer base url (The example below is referred to staging)
 LEGACY_CLUSTER_IPFS_URL="https://nft.storage.ipfscluster.io/api"
 
-# Indexer base url
-# 0 -> all request to Indexer
-# 100 -> All request to Pickup
-# values in between the balancer is applied (eg. 15 -> 15% of the request to Pickup, 85% to Indexer)
-BALANCER_RATE=10
```

Comment:
Removed as we no longer split requests between pickup and cluster, and the balancer code has gone.